scripts/pigmentcss-render-mui-demos.mjs

```diff
@@ -2,12 +2,14 @@ import path from 'path';
 import fse from 'fs-extra';
 import * as prettier from 'prettier';
 
-function capitalize(string) {
+function pascalCase(string) {
```

Comment:
`react-app-bar` was throwing, it resulted in `App-bar()` which is not a valid function name, I've converted to using pascal case.